src/passes/used.rs

```diff
@@ -64,6 +65,32 @@ impl Used {
             stack.push_func(f);
         }
 
+        // Initialization of imported memories or imported tables is a
```

Comment:
Note that this is arguably a bug in wasm-gc as-is today!

Reply:
> If a module imports a table/memory and has initializers for it, then we should consider that import live and retained to ensure initializers happen when the module is instantiated.

Yeah, that was an existing bug for sure